### ISSUE TITLE
v3 - Ability to upload custom member list

### ISSUE THREAD

**Issue by maartenhbe** [ZeusWPI/Gandalf#488]
After the Rudi Vranckx sale I was thinking that it would be nice to be able to link a custom member list for a specific ticket type.

My idea is to have an organisation create multiple "member lists" in their profile. For example, Politeia could have a default member list linked to enrollment, a "list" that would allow only UGent students to register and the ability to upload a csv in their profile that is listed as a seperate member list. On the ticket creation page one would be able to select one of those lists as target audience.

What do you think?

**Comment by nudded** [comment]
Good idea, i thought it would be best if they could do this in enrollment, since this is where all the data is stored. (it would require an update to the isengard api, to allow a subgroup).

**Comment by maartenhbe** [comment]
Wouldn't that make enrollment something very bloated with data?
